.github/workflows/typescript-untouched-files-check.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: ustaxcourt/ef-cms
-          ref: ${{ env.TARGET_BRANCH }}
+          ref: staging
```

Comment:
@cruzjone-flexion is this temp? or do want to merge this?

Reply:
Temp